client/src/modules/journal/journal.js

```diff
@@ -293,9 +293,13 @@ function JournalController(
     headerCellFilter : 'translate',
     visible : true,
   }, {
-    field : 'transaction_type_id',
+    field : 'transaction_type_text',
     displayName : 'FORM.LABELS.TRANSACTION_TYPE',
     headerCellFilter : 'translate',
+    cellTemplate :
```

Comment:
This works - a better solution is to use the translate filter:
```js
cellFilter : 'translate',
```

This will avoid creating a new template.